content/how-to/exchange/hosted_exchange_mailbox/index.md

```diff
@@ -0,0 +1,170 @@
+---
+permalink: hosted_exchange_mailbox
```

Comment:
I noticed in the other article we used underscore too. I would suggest we change to a dash instead here.  so `hosted-exchange-mailbox`. Rename the folder as well. Update any links or permalinks that point to this file as well.